src/App.js

```diff
@@ -0,0 +1,21 @@
+import { Route, Routes } from 'react-router-dom';
+import './App.css';
+import Books from './pages/Books';
+
+import Categories from './pages/Categories';
+// eslint-disable-next-line
```

Comment:
- [ ] As requested by @Gambit142 kindly remove the Eslint-disable statement from this file. The Eslint helps us write clean, maintainable, and scalable codes. Using Eslint-disable statements defeats this purpose.